src/index.html

```diff
@@ -11,8 +11,60 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The href attribute is pointing to a SCSS file. Typically, browsers cannot directly interpret SCSS files, so you should link to a compiled CSS file instead.